perfscale_regression_ci/scripts/scalability/replace_loadedworker_nodes_env.sh

```diff
@@ -0,0 +1,8 @@
+source ../custom_workload_env.sh
+
+export WORKLOAD="cluster-density"
+export INDEXING=true
+export METADATA_COLLECTION=true
+export METRICS_PROFILE="metrics-profiles/metrics-aggregated.yaml"
+export COMPARISON_CONFIG="clusterVersion.json nodeMasters-max.json nodeAggWorkers.json"
```

Comment:
COMPARISON_CONFIG and GEN_CSV should be added to this file to be used be all the workloads https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts/custom_workload_env.sh